server/app/forms/translation/ProgramTranslationForm.java

```diff
@@ -184,6 +209,29 @@ private ImmutableList<LocalizationUpdate.StatusUpdate> parseStatusUpdatesFromReq
         .collect(ImmutableList.toImmutableList());
   }
 
+  private ImmutableList<LocalizationUpdate.ScreenUpdate> parseScreenUpdatesFromRequest(
+      ImmutableList<Long> blockIds) {
+    return blockIds.stream()
+        .map(
+            blockId -> {
+              Optional<String> maybeBlockName = getStringFormField(localizedScreenName(blockId));
```

Comment:
**nit, not-blocking**: I think the rough naming convention we decided was to use `optionalName` rather then `maybeName` for optionals.

Reply:
done